internal/component/common/relabel/label_builder.go

```diff
@@ -0,0 +1,9 @@
+package relabel
+
+// LabelBuilder is an interface that can be used to change labels with relabel logic.
+type LabelBuilder interface {
+	Get(label string) string
```

Comment:
Should we add a comment that Get called with an unfound value returns a blank string? Or change signature to include found?